README.md

```diff
@@ -2,12 +2,23 @@ Ramble is a multi-platform experimentation framework that is capable of driving
 software installation, acquire input files, configure experiments, and extract
 results. It works on Linux, macOS, and many supercomputers.
 
+Ramble can be used to configure a variety of experiments for applications.
```

Comment:
I think it would be nice if the readme was more clearly able to communicate what is ramble / what problem it solves / why someone who randomly comes here should care. 

I think you almost do that, but something even more direct in the vain of "ramble is a python framework to increase benchmarking productivity and improve reproducibility.."

Reply:
How does it sound now?

Reply:
Much nice!